File: cpp/funkcje.cpp
/*
 * szablon.cpp
 */


#include <iostream>

using namespace std;

int dodaj(int a, int b)
{
        return a + b;
}

int roznica(int a, int b)
{
        return a - b;
}

int iloczyn(int a, int b)
{
        return a * b;
}

int iloraz(int a, int b)
{
        return a / b;
}

int main(int argc, char **argv)
{
	float a, b; //deklaracja zmiennej
    a = b = 0; //inicjacja zmiennej
    
    cout << "Podaj 1. liczbę: ";
    cin >> a;
    cout << a << endl;
    
    cout << "Podaj 2. liczbę: ";
    cin >> b;
    cout << b << endl;
    
    cout << "Suma: " << dodaj(a, b) << endl;
    cout << "Różnica: " << roznica(a, b) << endl;
    cout << "Iloczyn: " << iloczyn(a, b) << endl;
    cout << "Iloraz: " << iloraz(a, b) << endl;
    
    
    
	return 0;
}

File: Python/funkcje.py
#!/usr/bin/env python3
# -*- coding: utf-8 -*-
#
#  szablon.py

# zasięg zmiennych: lokalny, globalny

def dodaj(a, b):
    return a + b
    
def roznica(a, b):
    return a - b
    
def iloczyn(a, b):
    return a * b
    
def iloraz(a, b):
    return a / b

def main(args):
    #a = 0 # inicjacja zmiennnej
    a = int(input('Podaj 1. liczbę: '))
    print(a)
    b = int(input('Podaj 2. liczbę: '))
    print(b)
    
    print("Suma: ", dodaj(a, b))
    print("Różnica: ", roznica(a, b))
    print("Iloczyn: ", iloczyn(a, b))
    print("Iloraz: ", iloraz(a, b))
    
    return 0
    
# duck typing

if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))
